README.md

```diff
@@ -7,11 +7,11 @@ To complete this homework you should
 3. Create a new file called `FIRSTNAME_LASTNAME.md`
    - For example, if your name is Joe Smith then call your file `JOE_SMITH.md`
 4. In this file, write an introduction yourself. You should include:
-   1. Your name
-   2. Your favourite sport and why you like it
-   3. Your favourite food and why it's your favourite
-   4. A link to a video of your favourite music
-   5. An interesting fact about yourself - impress us!
+   Colette Sweeney
+   2. Your favourite sport and why you like it - Rugby as its fast and interesting
+   3. Your favourite food and why it's your favourite - Past and pesto Yum!
+   4. A link to a video of your favourite music - https://www.youtube.com/watch?v=-rrrof1TJc4
+   5. An interesting fact about yourself - impress us! I would like to run a marathon 🏃‍♀️
 5. `Commit` the file to your repository
```

Comment:
You need to put the the facts about yourself into a new file based on your name, so `COLETTE_SWEENEY.md` in this case. This is *not* the name of the commit - it's the name of the file itself, just as README.md exists in its own file. Then you could have a commit message of something like "add COLETTE_SWEEMEY.md" so that when reading it you (or someone else) can tell what that commit changes.